onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
I will propose a second solution as well after playing with apt-get a little.  We could use a stock java/ubuntu image and install the mesosphere repo as described [here](https://mesosphere.com/downloads/). Then install the proper version of mesos with:
```
apt-get install mesos=`apt-cache madison mesos | grep ${MESOS_RELEASE} | head -1 | cut -d'|' -f2 | cut -d' ' -f2`
```
`apt-cache madison mesos` displays all the versions of mesos i.e.
```
     mesos | 0.28.0-2.0.16.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.27.2-2.0.15.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.27.1-2.0.226.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.27.0-0.2.190.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.26.0-0.2.145.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.25.0-0.2.70.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.24.1-0.2.35.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.24.0-1.0.27.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.23.1-0.2.61.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.23.0-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.22.2-0.2.62.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.22.1-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.22.0-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.21.2-0.2.63.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.21.1-1.1.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.21.0-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.20.1-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.20.0-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.19.1-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.19.0-1.1.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
     mesos | 0.19.0-1.0.ubuntu1404 | http://repos.mesosphere.io/ubuntu/ trusty/main amd64 Packages
```